src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/custom.py

```diff
@@ -0,0 +1,135 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=inconsistent-return-statements
+
+
+# Namespace Region
+def cli_namespace_create(client, resource_group_name, namespace_name, location=None, tags=None, sku='Standard', capacity=None, is_auto_inflate_enabled=None, maximum_throughput_units=None):
+    from azure.mgmt.eventhub.models import EHNamespace, Sku
+    return client.create_or_update(
+        resource_group_name=resource_group_name,
+        namespace_name=namespace_name,
+        parameters=EHNamespace(
+            location=location,
+            tags=tags,
+            sku=Sku(sku, sku, capacity),
+            is_auto_inflate_enabled=is_auto_inflate_enabled,
+            maximum_throughput_units=maximum_throughput_units))
+
+
+def cli_namespace_update(instance, tags=None, sku='Standard', capacity=None, is_auto_inflate_enabled=None, maximum_throughput_units=None):
+
+    if tags:
+        instance.tags = tags
+
+    if sku:
+        instance.sku.name = sku
+        instance.sku.tier = sku
+
+    if capacity:
+        instance.sku.capacity = capacity
+
+    if is_auto_inflate_enabled:
+        instance.is_auto_inflate_enabled = is_auto_inflate_enabled
+
+    if maximum_throughput_units:
+        instance.maximum_throughput_units = maximum_throughput_units
+
+    return instance
+
+
+def cli_namespace_list(client, resource_group_name=None):
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name=resource_group_name)
+
+    if not resource_group_name:
+        return client.list()
```

Comment:
Just return `client.list()`. The second if statement is not needed.

Reply:
implemented